drgn_tools/task.py

```diff
@@ -161,3 +169,93 @@ def task_cpu(task: drgn.Object) -> int:
     if has_member(task, "cpu"):
         return task.cpu.value_()
     return task_thread_info(task).cpu.value_()
+
+
+def translate_nanoseconds(task: Object) -> str:
+    """
+    :returns: conversion of nanoseconds to [dd hh:mm:ss.ms] format
+    """
+    nanosecs = task_lastrun2now(task)
+    secs = nanosecs_to_secs(nanosecs)
+    time = str(datetime.timedelta(seconds=secs))
+    return time
+
+
+def get_pid(task: Object) -> int:
+    """
+    :returns: PID of the task
+    """
+    return task.pid.value_()
+
+
+def get_ps_state(task: Object) -> str:
+    """
+    :returns: state of the task
+     D    uninterruptible sleep
+     I    Idle kernel thread
+     R    running or runnable
+     S    interruptible sleep
+     T    stopped by job control signal
+     t    stopped by debugger during the tracing
+     X    dead (should never be seen)
+     Z    zombie
+    """
+    return task_state_to_char(task)
+
+
+def get_command(task: Object) -> str:
+    """
+    :returns: name of the command
+    """
+    return task.comm.string_().decode("utf-8")
+
+
+def show_ps_m_info(prog: Program) -> None:
+    """
+    Display task information in their last arrival order.
+    """
+    print(
+        "LAST_ARRIVAL".ljust(25)
+        + "STATE".ljust(18)
+        + "PID".ljust(20)
+        + "TASK".ljust(20)
+        + "CPU".ljust(15)
+        + "COMMAND".ljust(20)
+    )
+    tasks = []
+    for t in for_each_task(prog):
+        cpu = task_cpu(t)
+        pid = get_pid(t)
+        state = get_ps_state(t)
+        task = find_task(prog, pid)
+        command = get_command(t)
+        last_arrival = translate_nanoseconds(t)
+        nanosecs = task_lastrun2now(t)
+        tasks.append(
+            [
```

Comment:
Nit: make `tasks` a list of tuples, rather than a list of lists:
```suggestion
            (
```